test/_features/example_app/validation.feature

```diff
@@ -132,20 +133,23 @@ Feature: validations
     Then I should see the 'Select an appeal stage from the list' error
     Then I select 'appealStages' and '01. First Tier IAC Appeal - In Country Appeals'
     Then I click the 'Continue' button
+    Then I should be on the 'save-form' page showing 'Would you like to save your form?'
+    Then I continue to the next step
+    Then I should see the 'Select if you want to save your form or not' error
+    Then I should be on the 'save-form' page showing 'Would you like to save your form'
+    Then I check 'saveForm-no'
+    Then I click the 'Continue' button
+    Then I should be on the 'confirm' page showing 'Check your answers before submitting your application.'
     Then I continue to the next step
     Then I should see 'Application complete' on the page
     Then I click the 'Start again' button
     Then I should be on the 'landing-page' page showing 'Choose one of the options below and press continue.'
 
-  @complex_form @continue-saved-form
+  @complex_form @continue_saved_form
   Scenario: Partial Complex Form Submission Continuing A Saved Form
     Given I start the 'base' application journey
-    Then I continue to the next step
-    Then I should see the 'Select an option below and press continue' error
     Then I choose 'Complex form'
     Then I continue to the next step
-    Then I continue to the next step
```

Comment:
Line 147 and line 148 should not be removed here

Reply:
added both lines back